### PR TITLE
add cloud explainer to landing page

### PR DESCRIPTION
**Pull request by sw-yx**
## What does this PR do?

add cloud explainer to landing page

we are doing this because people think we aren't selling cloud yet - time to tweak the perception

### CODE REVIEW

**Review by flossypurse — APPROVED** [review]

File: src/pages/index.js [hunk 1]
```diff
@@ -5,168 +5,225 @@ import Link from '@docusaurus/Link';
...
+            product feedback.
+          </li>
+          <li>
+            <strong>Future improvements</strong>: to be discussed as needed
+          </li>
+        </ul>
+      </p>
+      <p>
+        Temporal Cloud is working with early design partners today. Sign up to
+        the waitlist and to receive updates!
```
Comment (flossypurse) [inline src/pages/index.js:175]:
```suggestion
        the waitlist and receive updates!
```

File: src/pages/index.js [hunk 2]
```diff
@@ -5,168 +5,225 @@ import Link from '@docusaurus/Link';
...
   );
 }
 
+function TemporalCloud() {
+  return (
+    <div className={styles.formFeature}>
+      <h2>☁️ Temporal Cloud</h2>
+      <p>
+        Temporal Cloud is a fully managed cloud offering from the company behind
+        Temporal's open source. We offer:
```
Comment (flossypurse) [inline src/pages/index.js:147]:
```suggestion
        What are we offering?
```

File: src/pages/index.js [hunk 3]
```diff
@@ -5,168 +5,225 @@ import Link from '@docusaurus/Link';
...
+        <ul>
+          <li>
+            <strong>Updates</strong>: Automatic updates with latest releases.
+          </li>
+          <li>
+            <strong>Experience</strong>: Have run this technology in production
```
Comment (flossypurse) [inline src/pages/index.js:153]:
```suggestion
            <strong>Experience</strong>: Our team has years of experience operating the technology in production environments.
```

File: src/pages/index.js [hunk 4]
```diff
@@ -5,168 +5,225 @@ import Link from '@docusaurus/Link';
...
+          <li>
+            <strong>Dependencies</strong>: No more managing dependencies like
+            Elasticsearch, Cassandra or MySQL.
+          </li>
+          <li>
+            <strong>Prioritization</strong>: Dedicated channels for support and
+            product feedback.
+          </li>
+          <li>
+            <strong>Future improvements</strong>: to be discussed as needed
```
Comment (flossypurse) [inline src/pages/index.js:169]: I think this one can be removed.
Doesn't really say much beyond "automatic updates" which is covered above.

**Review by flossypurse — APPROVED** [review]
Some suggestions to think about.

File: src/pages/index.js [hunk 5]
```diff
@@ -5,168 +5,224 @@ import Link from '@docusaurus/Link';
...
+          </li>
+        </ul>
```
Comment (sw-yx) [inline src/pages/index.js:170]:
```suggestion
          </li>
        </ul>
```

File: src/pages/index.js [hunk 6]
```diff
@@ -5,168 +5,221 @@ import Link from '@docusaurus/Link';
...
+        <ul>
+          <li>
+            <strong>Updates</strong>: Automatic updates with latest releases.
+          </li>
+          <li>
+            <strong>Experience</strong>: Our team has years of experience operating the technology in production environments.
+          </li>
+          <li>
+            <strong>Scale</strong>: Our design partners are
+           are highly valued publicly listed companies.
```
Comment (flossypurse) [inline src/pages/index.js:157]:
```suggestion
            <strong>Scale</strong>: Our design partners are highly valued publicly listed companies.
```

**Pull request merged**